dashboard/15-final/.env

```diff
@@ -1,3 +1,4 @@
+NEXTAUTH_SECRET=7XMu1K3gXGmKaVKW02Lo4Z8Vi2kkcFfpQUmt2yHMgFg=
```

Comment:
We shouldn't be committing any of these values, right? Shouldn't the whole file be deleted?